content/partner/gcp.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Cloud Native Infrastructure as Code for Google Cloud with Pulumi"
+date: 2018-09-01T09:37:50-07:00
+layout: "gcp"
+page_class: "partner"
+
+url: "/gcp"
+
+meta_title: "Cloud Native Infrastructure as Code for Google Cloud with Pulumi"
+meta_desc: "Programming Google Cloud with Pulumi for huge productivity gains, and a unified programming model for Devs and DevOps."
+meta_image: "assets/images/pulumi.png"
+
+topic: "Cloud Native Infrastructure as Code"
+hero_title: "Program the cloud with Pulumi and Google Cloud"
+hero_description: "Pulumi provides a cloud native programming model for Google Cloud to create containers, serverless functions, and infrastructure, enabling the delivery of Cloud Native Infrastructure as Code, using real programming languages. <br><br>Find out how to program the cloud with Pulumi and Google Cloud."
+hero_classes: "bg-purple white-text"
+hero_right_content: '<div class="code_container">
+    <div class="anim_headbar">
+        <div class="dot-container">
+            <div class="window-dot dot-red"></div>
+            <div class="window-dot dot-yellow"></div>
+            <div class="window-dot dot-green"></div>
+        </div>
+    </div>
+    <div class="anim_textbox code_display_textbox">
+        <div class="code-display">
+<pre><code class="javascript hljs">import * as k8s from "@pulumi/kubernetes";
```

Comment:
Just a note: In the short term, we'll want to get https://highlightjs.org setup so these code snippets are syntax highlighted. The blog also uses highlight.js for highlighting code snippets, so it'd also highlight existing code snippets in blog posts (until they've been converted to markdown and get Hugo's built-in [Chroma-based highlighting](https://gohugo.io/content-management/syntax-highlighting/)).

Reply:
cc: @chrsmith as he mentioned he's converting the blog posts over to use Hugo's [`highlight` shortcode](https://gohugo.io/content-management/syntax-highlighting/#highlight-shortcode). Although, I suggest we standardize on [Code Fences](https://gohugo.io/content-management/syntax-highlighting/#highlight-in-code-fences) long-term for most code snippets (unless we need something more specialized like line numbers and line highlighting), as that's what's used everywhere else in existing docs content and in API doc comments in package repos.